docs/helm-deployment-aws_kops.md

```diff
@@ -339,7 +339,7 @@ export EFS_SERVER=<EFS_ID>.efs.<AWS-REGION>.amazonaws.com
 ```
 ### Creating a Docker registry pull secret
 
-Since we need to use protected Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images.
+Since we need to use protected Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images. Alfresco customers can request their credentials by logging a ticket at https://support.alfresco.com.
```

Comment:
I'm not entirely sure about this addition. AFAIK if it is a paying customer, the credentials should be generated automatically for their account in salesforce.

Reply:
That's correct, but customers still need to request their credentials in this way. I copied the change from the master branch after a PR from Support.